packages/blade/src/_helpers/storybook/ComponentStatusTable.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import styled from 'styled-components';
+import type { IconComponent } from '../../components/Icons';
+import {
+  AlertCircleIcon,
+  EditIcon,
+  CheckIcon,
+  ClockIcon,
+  LoaderIcon,
+} from '../../components/Icons';
+import { Text } from '~components/Typography';
+import { makeSpace } from '~utils';
+import Box from '~components/Box';
+import type { BadgeProps } from '~components/Badge';
+import { Badge } from '~components/Badge';
+import { Link } from '~components/Link';
+
+type ComponentStatuses =
+  | 'released'
+  | 'in-api-spec'
+  | 'in-development'
+  | 'in-design'
+  | 'deprecated'
+  | `planned-Q${1 | 2 | 3 | 4}`;
+
+type ComponentStatusData = {
+  name: string;
+  status: ComponentStatuses;
+  releasedIn?: string;
+}[];
+
+const componentData: ComponentStatusData = [
```

Comment:
lets sort this as discussed yesterday.

Reply:
Done. Note: I could've done category wise grouping with the new storybookLink url in the data but didn't because the sorting logic was getting a bit complex, so this is manual for now. 

Anyways when we add the pending component data the sorting might get messed up.

Reply:
any plans to automate this? Probably should not end up opening new PR's just to update this object.

Reply:
> any plans to automate this

Not right now, components/roadmaps won't change that often so not planning to automate this. 

Tho a better option might be to have a google sheet or a github gist and fetch the data from there so that we can make changes faster without committing to repo everytime we make a change.